pom.xml

```diff
@@ -58,6 +58,7 @@ under the License.
     <protobuf.version>3.25.5</protobuf.version>
     <parquet.version>1.13.1</parquet.version>
     <parquet.maven.scope>provided</parquet.maven.scope>
+    <hadoop.version>3.3.4</hadoop.version>
```

Comment:
Does it make sense to use the same hadoop version as what Spark uses? I.e. should we use maven profile to change versions?

Reply:
Just for testing, I think we don't need to specify hadoop version for each spark version, and we introduced a shaded hadoop client which doesn't seem to cause conflicts.